### ISSUE TITLE
Installation on Windows

### ISSUE THREAD

**Issue by waczkor** [zk00006/OpenTLD#18]
Hy All!

My windows decided to give me some BSOD, so I had to get a new win os. 
So the point is...
I have win7 x 64, Visual Studio 2010, and if I use Matlab 2010a with the patch, shall I use OpenCV2.2 for Win32?
(Anyway making OpenCv compatible for win7x64 is kinda messy for me. ) 

Thanks for the answer.

**Comment by TheMrtN** [comment]
This is not the right place to ask questions about your installation. It is ment for issues with the code itself.
A good place to discuss installation issues is the Google Group:
https://groups.google.com/forum/#!forum/opentld
Please first search through the group before posting a new thread.

To answer your question: here's an example of someone who was able to compile OpenTLD for 32-bit:
https://groups.google.com/d/msg/opentld/Lu51xn0txZI/Hgkv73xNvTYJ

**Comment by waczkor** [comment]
I'm still waiting for the Google Group membership, that's why I added this question here. Sorry.

**Comment by rayryeng** [comment]
If you use the patch with OpenCV 2.2 it'll work.  I actually gave up after a certain point because the libraries weren't compiling for me.  I just installed the 32-bit version of MATLAB... easiest thing for me.

**Comment by waczkor** [comment]
I use the patch but it does not work. :$

On 2011.07.05. 18:34, "rayryeng" <
reply@reply.github.com>
wrote:

If you use the patch with OpenCV 2.2 it'll work.  I actually gave up after a
certain point because the libraries weren't compiling for me.  I just
installed the 32-bit version of MATLAB... easiest thing for me.

##

**Comment by rayryeng** [comment]
I don't know buddy.... install 32-bit MATLAB instead, then try doing it.  That's what I did.

Good luck,
- Ray.

**Comment by waczkor** [comment]
Is it normal , when i push any button, the running stops?

**Comment by rayryeng** [comment]
Yup, a callback was written to do that. You can comment out that code if you
like.

**Comment by waczkor** [comment]
I'm sorry, I can't find the callback..
could you tell me where it is?

**Comment by bilderbuchi** [comment]
https://github.com/zk00006/OpenTLD/blob/master/tld/tldExample.m#L27

but please try to use the google group for those questions. send zdenek a message on here maybe to speed up the inclusion in the group if you're not already in.

**Comment by njgongkai** [comment]
hello ，I'm a freshman on computer_vision.I have watched the flash of 
TLD.
but,I can't download code,oh my god!
Is there anybody give a hand to me?
Thank you !
myemail@  njgngkai@gmail.com

**Comment by rayryeng** [comment]
https://github.com/zk00006/OpenTLD/zipball/master
